src/uu/realpath/src/realpath.rs

```diff
@@ -76,6 +79,18 @@ pub fn uu_app() -> App<'static, 'static> {
                 .long(OPT_ZERO)
                 .help("Separate output filenames with \\0 rather than newline"),
         )
+        .arg(
+            Arg::with_name(OPT_LOGICAL)
+                .short("L")
+                .long(OPT_LOGICAL)
+                .help("resolve '..' components before symlinks"),
+        )
+        .arg(
+            Arg::with_name(OPT_PHYSICAL)
+                .short("P")
+                .long(OPT_PHYSICAL)
+                .help("resolve symlinks as encountered (default)"),
+        )
```

Comment:
I think these options should override each other. See [`overrides_with`](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.overrides_with)